### PR TITLE
button created

### PR DESCRIPTION
**Pull request by danielnottingham**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class User < ApplicationRecord
+    validates :name, :phone, presence: true
+    validates :name, :phone, format: { with: /\A[a-zA-Z]+\z/}
```
Comment (edimossilva) [inline app/models/user.rb:3]: should name and phone have the same format validation?

File: spec/requests/users_request_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (edimossilva) [inline spec/requests/users_request_spec.rb:1]: from now, all your code needs to have test coverage, and it has to be 100% alright?

**Review by edimossilva — CHANGES_REQUESTED** [review]

**Comment by edimossilva** [comment]
please update the related issue and tag the issue into the PR description

File: spec/requests/users_request_spec.rb (hunk 2 above)
Comment (danielnottingham) [reply spec/requests/users_request_spec.rb:1]: okay

File: app/models/user.rb (hunk 1 above)
Comment (danielnottingham) [reply app/models/user.rb:3]: not necessary, I wanted to validate both fields as a string

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: [:show, :destroy]
+
```
[the remark this answers is not in the document]
Comment (danielnottingham) [reply app/controllers/users_controller.rb:3]: not necessary, removed.

File: spec/requests/users_request_spec.rb (hunk 2 above)
Comment (danielnottingham) [reply spec/requests/users_request_spec.rb:1]: Okay. I tested it and it was 100%
I'm learning new ways to keep the code fully covered

File: app/models/user.rb (hunk 1 above)
Comment (edimossilva) [reply app/models/user.rb:3]: please specify the validations in the issue, so do  you intend to allow names to be set into phone field?

File: spec/helpers/users_helper_spec.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (edimossilva) [inline spec/helpers/users_helper_spec.rb:1]: remove this file?

File: spec/requests/users_request_spec.rb (hunk 2 above)
Comment (edimossilva) [reply spec/requests/users_request_spec.rb:1]: even the controllers?

Comment (danielnottingham) [reply spec/requests/users_request_spec.rb:1]: no, i will add

File: spec/helpers/users_helper_spec.rb (hunk 4 above)
Comment (danielnottingham) [reply spec/helpers/users_helper_spec.rb:1]: removed

Comment (edimossilva) [reply spec/helpers/users_helper_spec.rb:1]: when you are 100% sure that the requirement has been done you can "Resolve conversation"

**Comment by edimossilva** [comment]
Also, add a PrintScreen with the test report to the PR description

File: app/controllers/users_controller.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (edimossilva) [reply app/controllers/users_controller.rb:2]: Mark this as resolved

File: app/helpers/users_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```
Comment (edimossilva) [inline app/helpers/users_helper.rb:1]: Remove this

File: config/routes.rb [hunk 6]
```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  get 'users/new'
+  post 'users/new', action: :create, controller: 'users'
```
Comment (edimossilva) [inline config/routes.rb:4]: Instrad If naming It as 'users/new' name It as 'users/create'. It is a Better convention

File: spec/requests/users_request_spec.rb (hunk 2 above)
Comment (edimossilva) [reply spec/requests/users_request_spec.rb:1]: Mark as resolved

**Review by edimossilva — APPROVED** [review]

**Comment by edimossilva** [comment]
> Also, add a PrintScreen with the test report to the PR description

@danielnottingham , did you forget this?

**Pull request merged**